python_docs_theme/static/pydoctheme.css

```diff
@@ -242,7 +242,74 @@ div.body pre {
     border: 1px solid #ac9;
 }
 
-div.body div.admonition,
+/* Admonitions */
+:root {
+    --admonition-background: #eee;
+    --admonition-border: #ccc;
+    --attention-background: #bbddff5c;
+    --attention-border: #0000ff36;
+    --caution-background: #ffc;
+    --caution-border: #dddd66;
+    --danger-background: #ffe4e4;
+    --danger-border: red;
+    --error-background: #ffe4e4;
+    --error-border: red;
+    --hint-background: #bfc;
```

Comment:
`#bfc` is a bit vibrant for my tastes.  I'm also curious why attention has alpha specified?  Where did these colors come from?

Reply:
For new colours, I started with the ones in the PEPs repo. 

I then adjusted them using Chrome's dev tools to shift the colour a bit, but mostly the alpha, so the contrast was high enough to meet WCAG AA. It was mainly the hyperlink that needed the adjustment. 

Happy to change any of these. Any suggestions? Or I can try toning down the `#bfc` a bit?

We can also replace the blue and green with the default grey, if we want to fine tune colours in another PR. 

We might also want to replace some of the other default greys in followup PRs, compare some of the other themes' Kitchen Sink demos.

Reply:
I'd change `#bfc` to `#dfd`, it gets the point across.

Reply:
How's this?

![image](https://github.com/python/python-docs-theme/assets/1324225/4a135070-59da-4083-b4e4-460df8379e5a)

Reply:
I like it. 👍